src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,100 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_AL_SIZE = 10;
+    private static final double CAPACITY_GROWTH_VALUE = 1.5;
+    private static final int ZERO_NUMBER = 0;
```

Comment:
The constant 'ZERO_NUMBER' is unnecessary since zero is not a magic number and its use in the context of 'size' is self-explanatory. Consider removing it and using 0 directly.